composer.json

```diff
@@ -32,24 +32,23 @@
     },
     "require": {
         "php": "~8.1.0 || ~8.2.0 || ~8.3.0 || ~8.4.0",
+        "laminas/laminas-eventmanager": "^3.14",
+        "laminas/laminas-modulemanager": "^2.17",
         "laminas/laminas-serializer": "^3.0",
-        "laminas/laminas-servicemanager": "^4.0",
+        "laminas/laminas-servicemanager": "^4.4.0",
         "laminas/laminas-stdlib": "^3.20",
+        "psr/container": "^1.1",
```

Comment:
Is `psr/container@2.0` installable? i.e. can we require `^1.0 || ^2.0` and check in CI that it's getting installed on lowest/latest deps?

Edit… The standalone container/pluginmanager will probably need updated types to co-exist with container 2. Probably better off looking at that in a separate patch unless I'm wrong.

Reply:
Thanks! I’d be happy to give that a try, but I agree it’s probably best to look at it in a separate patch to keep things focused.